ng/src/web/pages/usersettings/dialog.js

```diff
@@ -0,0 +1,693 @@
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('CVE Filter')} titleSize="3">
+                  <Select2
+                    name="cvefilter"
+                    value={cvefilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('NVT Filter')} titleSize="3">
+                  <Select2
+                    name="nvtfilter"
+                    value={nvtfilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('OVAL Filter')} titleSize="3">
+                  <Select2
+                    name="ovalfilter"
+                    value={ovalfilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('CERT-Bund Filter')} titleSize="3">
+                  <Select2
+                    name="certbundfilter"
+                    value={certbundfilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('DFN-CERT Filter')} titleSize="3">
+                  <Select2
+                    name="dfncertfilter"
+                    value={dfncertfilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+
+                <FormGroup title={_('All SecInfo Filter')} titleSize="3">
+                  <Select2
+                    name="allsecinfofilter"
+                    value={allsecinfofilter}
+                    size="30"
+                    onChange={onValueChange}>
+                    {render_options(filtersList.secinfo)}
+                  </Select2>
+                </FormGroup>
+              </div>
+            }
+
+            <h2>{_('Miscellaneous Settings')}</h2>
+
+            <FormGroup title={_('Auto Cache Rebuild')} titleSize="3">
+              <Checkbox
+                name="autocacherebuild"
+                checked={parse_yesno(autocacherebuild) === YES_VALUE}
+                checkedValue={YES_VALUE}
+                unCheckedValue={NO_VALUE}
+                onChange={onValueChange}/>
+            </FormGroup>
+          </Layout>
+        );
+      }}
+    </SaveDialog>
+  );
+};
+
+UserSettingsDialog.propTypes = {
```

Comment:
Please add proptypes for your props:
- visible
- onClose
- onSave
- data
- optionLists
- capabilities